src/deeprank_gnn/predict.py

```diff
@@ -183,7 +229,8 @@ def get_embedding(fasta_file: Path, output_dir: Path) -> List[Path]:
     log.info("#" * 80)
 
     # Load model and alphabet
-    model, alphabet = pretrained.load_model_and_alphabet(ESM_MODEL)
+    esm_path = fetch_weights()
+    model, alphabet = pretrained.load_model_and_alphabet(esm_path)
```

Comment:
annoying nitpick here; if a variable has `path` on the name I would expect it to be of type `Path`, not `string` - but I see this `load_model_and_alphabet` uses a `string` anyway: https://github.com/facebookresearch/esm/blob/2b369911bb5b4b0dda914521b9475cad1656b2ac/esm/pretrained.py#L24-L28